tests/tensorflow/nightly/base.libsonnet

```diff
@@ -0,0 +1,195 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local utils = import 'templates/utils.libsonnet';
+local volumes = import 'templates/volumes.libsonnet';
+
+{
+  BaseTpuVmTest:: {
```

Comment:
This file essentially copies [experimental.libsonnet](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/experimental.libsonnet). These classes should instead override `experimental.BaseTpuVmTest` where necessary.

Reply:
Yeah I tried to copy and separate out tensorflow tests. But I think thats not a good idea. I removed the copy and change it to experimental.BaseTpuVmTest. Thanks!